tests/integration/requirements.txt

```diff
@@ -1,3 +1,11 @@
 # Our code is based on the AWS SDKs
 boto3
 botocore
+
```

Comment:
Are these needed for this role's integration tests?

Reply:
Oh, it doesn't. Thank you